nominees/period-dignity-bristol.json

```diff
@@ -0,0 +1,58 @@
+{
+  "name": "Period Dignity Bristol",
+  "description": "A map app to connect people with free menstrual products",
+  "website": "https://www.periodfriendlybristol.org/",
+  "license": [
+    {
+      "spdx": "GPL-3.0",
+      "licenseURL": "https://github.com/PeriodFriendlyBristol/period-dignity-app-client/blob/master/LICENSE"
+    },
+    {
+      "spdx": "GPL-3.0",
+      "licenseURL": "https://github.com/PeriodFriendlyBristol/period-dignity-app-server/blob/master/LICENSE"
+    },
+    {
+      "spdx": "GPL-3.0",
+      "licenseURL": "https://github.com/PeriodFriendlyBristol/period-dignity-app-wrapper/blob/master/LICENSE"
+    }
+  ],
+  "SDGs": [
+    {
+      "SDGNumber": 5,
+      "evidenceURL": "https://www.bristolpost.co.uk/news/bristol-news/period-poverty-period-friendly-bristol-3657690"
+    },
+    {
+      "SDGNumber": 6,
+      "evidenceURL": "https://www.bristolpost.co.uk/news/bristol-news/period-poverty-period-friendly-bristol-3657690"
+    },
+    {
+      "SDGNumber": 10,
+      "evidenceURL": "https://www.bristolpost.co.uk/news/bristol-news/period-poverty-period-friendly-bristol-3657690"
+    },
+    {
+      "SDGNumber": 11,
+      "evidenceURL": "https://www.bristolpost.co.uk/news/bristol-news/period-poverty-period-friendly-bristol-3657690"
+    },
+    {
+      "SDGNumber": 17,
+      "evidenceURL": "https://www.bristolpost.co.uk/news/bristol-news/period-poverty-period-friendly-bristol-3657690"
+    }
+  ],
+  "sectors": [
+    "Gender and Minority Groups",
+    "Sustainable Cities",
+    "Water and Sanitation"
+  ],
+  "type": [
+    "software"
+  ],
+  "repositoryURL": "https://github.com/PeriodFriendlyBristol",
```

Comment:
@nathanbaleeta this is a GitHub org, not a repo. We need to choose a repo representative of this project, and just include one so that other parts of the site won't break, as we expect a repo URL here.

Reply:
Chose the [backend repo](https://github.com/PeriodFriendlyBristol/period-dignity-app-server/) to be representative of the project as documented in commit 5db5ad8.